internal/Picker/Picker.js

```diff
@@ -76,8 +77,6 @@ const forwardBlur = forward('onBlur'),
 	forwardKeyUp = forward('onKeyUp'),
 	forwardWheel = forward('onWheel');
 
-const allowedClassNames = ['picker', 'valueWrapper', 'joined', 'horizontal', 'vertical'];
```

Comment:
We should not remove this. It should be passed to `usePublicClassNames` instead of `publicClassNames: true`.
This is the allowed public class names so it should not be changed.

Reply:
This change causes some screenshot tests to fail. For instance (text not centered):
![image](https://user-images.githubusercontent.com/63341832/208705575-09fc38ba-a2ed-42fa-b597-e8aecc0e2fad.png)
If I switch it back to publicClassNames: true it looks ok.